src/utils/devtools.ts

```diff
@@ -32,58 +34,69 @@ export const devtools = <T extends object>(proxyObject: T, name?: string) => {
   let isTimeTraveling = false
   const devtools = extension.connect({ name })
   const unsub1 = subscribe(proxyObject, (ops) => {
-    const action = ops.map(([op, path]) => `${op}:${path.join('.')}`).join(', ')
+    const action = ops
+      .flatMap(([op, path]) =>
+        path.some((x) => typeof x === 'symbol')
+          ? []
+          : [`${op}:${path.join('.')}`]
+      )
+      .join(', ')
+
+    if (!action) {
+      return
+    }
```

Comment:
Why do you need to return?

Reply:
Because we don't want the log in the redux devtools, it's just an empty log.

Reply:
Oh, I missed that `action` can be an empty string.